up42/viztools.py

```diff
@@ -101,11 +103,14 @@ def plot_results(
         else:
             axs = [axs]
 
+        if len(bands) != 3:
+            if len(bands) == 1:
+                bands = bands * 3  # plot as grayband
```

Comment:
If there is only 1 band, this line duplicates it 3 times? Is this necessary?

Reply:
Yeah that is interesting, not neccessary, but more conveniant to use. If you read just one band, the rasterio show/plot function will apply the default colormap, so the image will be green/blue. Single band should be grayscale imho, so to not have to expose the colormap I thought this would be the best option. Will check if I can also just disable the colormap in case of a single band.

Reply:
Using cmap now, more efficient, thanks.